installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -10,16 +10,12 @@ In {product-title} version {product-version}, you can install a customized clust
 
 == Prerequisites
 
-* Review details about the
-xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
-processes.
-** Verify that {product-title} {product-version} is compatible with your {rh-openstack} version by using the "Supported platforms for OpenShift clusters" section. You can also compare platform support across different versions by viewing the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
-
-* Verify that your network configuration does not rely on a provider network. Provider networks are not supported.
-
-* Have a storage service installed in {rh-openstack}, like block storage (Cinder) or object storage (Swift). Object storage is the recommended storage technology for {product-title} registry cluster deployment. For more information, see xref:../../scalability_and_performance/optimizing-storage.adoc#optimizing-storage[Optimizing storage].
-
-* Have metadata service enabled in {rh-openstack}
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+** You verified that {product-title} {product-version} is compatible with your {rh-openstack} version by using the xref:../../architecture/architecture-installation.adoc#supported-platforms-for-openshift-clusters_architecture-installation[Supported platforms for OpenShift clusters] section. You can also compare platform support across different versions by viewing the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
```

Comment:
I've seen the `reviewed details` and `read the documentation` bullets in that order so many time it seems odd to have this one in between. Not that it's wrong....

Reply:
I noticed that too, but thought that I'd keep the existing structure of the prerequisites in this subsection as is and focus on the parallelism in this PR.